### ISSUE TITLE
Does this polyfill cause a stack overflow in IE11?

### ISSUE THREAD

**Issue by morewry** [goatslacker/es-symbol#1]
I have an issue (https://github.com/webcomponents/webcomponentsjs/issues/972) filed with the Web Components polyfills that appear to use this module to polyfill `Symbol`.

The issue is that using the Web Components polyfills to get Web Components working in IE 11 causes a stack overflow error.

![Out of stack space / Stack overflow](https://user-images.githubusercontent.com/1043291/42978883-3d8692dc-8b84-11e8-8160-ea80d3967aae.png)

![symbol.js defProp](https://user-images.githubusercontent.com/1043291/42978887-41cf4960-8b84-11e8-9c63-e314fc6da923.png)

I found that loading the `Symbol` polyfill from [core-js](https://github.com/zloirock/core-js) before the Web Component polyfills got things working as expected.

What appears to be fixing the problem is that a different `Symbol` polyfill gets loaded first. It creates `window.Symbol`. Then the feature is detected in `@webcomponents/webcomponentsjs/src/symbol.js`, so this `Symbol` polyfill (`es-symbol/dist/symbol.js`) doesn't get used.

**Comment by goatslacker** [comment]
Interesting, I haven't found this to be the case but I can check. Do you happen to have a code example that causes the stack overflow?

**Comment by morewry** [comment]
No, I'm sorry. I didn't get to the point where I was able to identify the exact circumstances that triggered the issue. I know it doesn't happen under all circumstances, and I believe it comes from utilization in [webcomponentsjs](https://github.com/webcomponents/webcomponentsjs), of which I am a user, not a sufficiently well-versed developer.

The maintainers of [webcomponentsjs](https://github.com/webcomponents/webcomponentsjs) have updated it since and those changes have at least alleviated the problem. Perhaps https://github.com/webcomponents/webcomponentsjs/pull/982 (the alleviating pull request) or @azakus (the contributor who made the PR) can help provide some more information about what circumstances conspired to cause the stack overflow and help you determine whether those circumstances should lead to any changes here.

I opened this issue because I found that the `Symbol` polyfill from [core-js](https://github.com/zloirock/core-js) did not have the same problem, which suggested to me that there _might_ be a way to improve the robustness of this polyfill.

**Comment by fclmman** [comment]
So could anyone get it work properly? using "@webcomponents/webcomponentsjs": "^2.2.7", still have out of stack space issue
